socorro/external/boto/crashstorage.py

```diff
@@ -292,6 +292,20 @@ class TelemetryBotoS3CrashStorage(BotoS3CrashStorage):
         reference_value_from='resource.elasticsearch',
     )
 
+    required_config.add_option(
+        'keybuilder_class',
```

Comment:
I don't understand what you're doing here. The keybuilder class is part of the connection context--not the crash storage. I don't see how the changes you're making has an effect.